tap_aggregator/src/aggregator.rs

```diff
@@ -62,8 +62,10 @@ pub fn check_and_aggregate_receipts(
         }
     }
 
+    let receipts = receipts.into_iter().map(Into::into).collect::<Vec<_>>();
```

Comment:
No big deal but here is one place I'd like to see what type it's being coerced to 😄